jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -29,6 +29,14 @@ fi
 
 [[ -z ${INSTALL_C17_COMPILER} ]] && INSTALL_C17_COMPILER=false
 
+# Bionic|Focal builds were affected by a "gpg: keyserver receive failed" in apt-key execution
+# that poisoned a lot of docker cache in different builds and nodes. Force invalidation
+# during a couple of month to rotate images
+if [[ "${DISTRO}" == 'bionic' || "${DISTRO}" == 'focal' ]] && \
+   [[ "$(date '+%s')" -lt "$(date -d '+60 days' '+%s')" ]]; then
```

Comment:
will this always be 60 days in the future instead of 60 days from a fixed date?

Reply:
I just expect to see a specific date like `2023-12-31` or `2024-01-10` or something

Reply:
eeeeeeeeeek :sweat: do not code late on Fridays. [33e8fe3](https://github.com/gazebo-tooling/release-tools/pull/1067/commits/33e8fe33af6b23b566264fbe90271296d01a2170) sorry for that